lib/internationalization/korean_handler.fz

```diff
@@ -0,0 +1,34 @@
+# This file is part of the Fuzion language implementation.
+#
+# The Fuzion language implementation is free software: you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as published
+# by the Free Software Foundation, version 3 of the License.
+#
+# The Fuzion language implementation is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+# License for more details.
+#
+# You should have received a copy of the GNU General Public License along with The
+# Fuzion language implementation.  If not, see <https://www.gnu.org/licenses/>.
+
+
+# -----------------------------------------------------------------------
+#
+#  Tokiwa Software GmbH, Germany
+#
+#  Source code of Fuzion standard library feature internationalization.korean_handler
+#
+# -----------------------------------------------------------------------
+
+# korean_handler -- provides data for korean format
+#
+public korean_handler : int_handler is
+
+    #format date using the most common korean formatting rules, see https://en.wikipedia.org/wiki/Date_and_time_notation_in_South_Korea
```

Comment:
# format date using the most common Korean formatting rules, see
	    # https://en.wikipedia.org/wiki/Date_and_time_notation_in_South_Korea